modules/rosa-policy-identity-access-management.adoc

```diff
@@ -5,10 +5,11 @@
 
 [id="rosa-policy-identity-access-management_{context}"]
 = Identity and access management
-
-
 Most access by Red Hat site reliability engineering (SRE) teams is done using cluster Operators through automated configuration management.
 
+=== Subprocessor
+For a list of the available subprocessors, see the link:https://access.redhat.com/articles/5528091 [Red Hat Subprocessor List on the Customer Portal].
```

Comment:
There is an extra space after the link URL here, so the link isn't rendering properly